### PR TITLE
Adding ClusterServiceVersion (CSV) for 2.0.0

### PR DESCRIPTION
**Pull request by piyush-nimbalkar**
**What this PR does / why we need it**:
This PR includes following changes. Not all changes will be required when we cut a branch from master every time.
- Adding 2.0.0 CSV (required so master can point to new operator version)
- Updating everywhere to point to portworx/px-operator image
- Cherrypick from 1.4.5 for released image update

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: deploy/olm-catalog/portworx/1.5.0/portworxoperator.v1.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     capabilities: Auto Pilot
     categories: "Storage"
     description: Cloud native storage solution for production workloads
-    containerImage: openstorage/openstorage-operator:master
+    containerImage: portworx/px-operator:1.5.0-dev
```
Comment (pure-jliao) [inline deploy/olm-catalog/portworx/1.5.0/portworxoperator.v1.5.0.clusterserviceversion.yaml:10]: here we are not using master tag?

File: deploy/olm-catalog/portworx/1.5.0/portworxoperator.v1.5.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -206,7 +206,7 @@ spec:
                     topologyKey: kubernetes.io/hostname
               containers:
               - name: portworx-operator
-                image: openstorage/openstorage-operator:master
+                image: portworx/px-operator:1.5.0-dev
```
Comment (pure-jliao) [inline deploy/olm-catalog/portworx/1.5.0/portworxoperator.v1.5.0.clusterserviceversion.yaml:209]: same here

File: deploy/olm-catalog/portworx/1.5.0/portworxoperator.v1.5.0.clusterserviceversion.yaml (hunk 1 above)
Comment (piyush-nimbalkar) [reply deploy/olm-catalog/portworx/1.5.0/portworxoperator.v1.5.0.clusterserviceversion.yaml:10]: This is the 1.5.0 CSV. Here we would be using 1.5.0-dev.
The master tag will be used in the 2.0.0 CSV that is added in this PR - https://github.com/libopenstorage/operator/pull/289/files#diff-0b2a9ce703ba4233fbacb6e85b3d8ddb3c927e87365a57427bcf8088d53c960aR10

**Review by haibinxie — APPROVED** [review]
LGTM

**Pull request merged**